src/core/server/base.js

```diff
@@ -259,16 +273,19 @@ export function runServer({ listen = true, app = appName } = {}) {
         const routes = require(`${app}/routes`).default;
         const createStore = require(`${app}/store`).default;
         /* eslint-enable global-require, import/no-dynamic-require */
-        const server = baseServer(routes, createStore, { appInstanceName: app });
+        const server = baseServer(
+          routes, createStore, { appInstanceName: app });
         if (listen === true) {
           server.listen(port, host, (err) => {
             if (err) {
               reject(err);
             }
-            log.info(`🔥  Addons-frontend server is running [ENV:${env}] [APP:${app}] ` +
-                     `[isDevelopment:${isDevelopment}] [isDeployed:${isDeployed}] ` +
-                     `[apiHost:${config.get('apiHost')}] [apiPath:${config.get('apiPath')}]`);
-            log.info(`👁  Open your browser at http://${host}:${port} to view it.`);
```

Comment:
They are a lot easier to read under 80 and it still outputs the same:

<img width="802" alt="screenshot 2016-12-22 23 37 15" src="https://cloud.githubusercontent.com/assets/90871/21443779/9ccb7d60-c89f-11e6-8c61-d7e013e104e4.png">
vs
<img width="821" alt="screenshot 2016-12-22 23 37 07" src="https://cloud.githubusercontent.com/assets/90871/21443781/a0fad9da-c89f-11e6-9c27-a4ada411c4b9.png">